extension/extension/background.js

```diff
@@ -189,8 +190,13 @@ function websocket_logger(server) {
     }
 
     return function(data) {
-        queue.push(data);
-        dequeue();
+        switch (authStatus) {
```

Comment:
How are the 2 other cases handled? Currently we push items to the queue if we receive `allow` or `deny_for_2_days`. We should handle each one individually.

We probably want the default case to happen if we receive an unsupported case. There's a chance the server gets updated to include a new case/keyword, but the clients haven't updated the extension, so they are not sure how to handle it yet.

Reply:
I have handled `deny_for_2_days` while allow is the default along with any future unsupported cases from the server